sdks/java/core/pom.xml

```diff
@@ -429,6 +429,12 @@
     </dependency>
 
     <dependency>
+      <groupId>com.google.code.findbugs</groupId>
```

Comment:
Yes, my findbugs PR is now in, which switches all dependencies to use com.google.code.findbugs:annotations instead of jsr305. 

If you still get unused/undeclared dependency errors after switching, run mvn dependecy:tree to see if jsr305 is being brought in transitively; you may need to explicitly exclude it.